src/modules/audio/source.c

```diff
@@ -274,15 +283,15 @@ void lovrSourceStop(Source* source) {
 
     case SOURCE_STREAM: {
 
+      // Stop the source
+      alSourceStop(source->id);
+      alSourcei(source->id, AL_BUFFER, AL_NONE);
+
       // Empty the buffers
       int count = 0;
       alGetSourcei(source->id, AL_BUFFERS_QUEUED, &count);
       alSourceUnqueueBuffers(source->id, count, NULL);
```

Comment:
I am confused how this line is working if the other one in `source.c` didn't work.  Oh well, it can be a future investigation, I really want to get this merged!